helm/app-operator/templates/configmap.yaml

```diff
@@ -16,6 +16,8 @@ data:
     service:
       app:
         unique: {{ include "resource.app.unique" . }}
+        watchNamespace: '{{ .Values.app.watchNamespace }}'
+        workloadClusterID: '{{ .Values.app.workloadClusterID }}'
```

Comment:
When watching app CRs in the org namespace these values will be set by cluster-apps-operator.